docs/how_to/chain_python.md

```diff
@@ -0,0 +1,42 @@
+# Branch a pipeline in Python
+
+:::{admonition} What does this guide solve?
+:class: important
+When working with Lumen in Python, you can build branching pipelines that allow you have different processing steps and views on the same source data. For instance, you may want to display a table of filtered data alongside a view of aggregated data.
+:::
+
+## Overview
+The primary tool to create a branch of a Lumen pipeline is to use the `pipeline.chain` method. By chaining the `Pipeline` we can apply new processing steps while retaining the shared computations up to that point.
+
+
+## Initiating the pipeline
+Let's start by creating a pipeline up to a branching point. See the [Build a dashboard in Python](ht_pipeline_python) how-to guide for a walkthrough of these initial steps.
+
+```python
+from lumen.pipeline import Pipeline
+from lumen.sources import FileSource
+
+data_url = 'https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv'
+
+pipeline = Pipeline(source=FileSource(tables={'penguins': data_url}), table='penguins')
+
+pipeline.add_filter('widget', field='species')
+pipeline.add_filter('widget', field='island')
+pipeline.add_filter('widget', field='sex')
+pipeline.add_filter('widget', field='year')
+
+pipeline.data
+```
+![data preview](../_static/pipeline_data.png)
```

Comment:
Can we remove the screenshots here? I'll add the pyodide rendering extension in a separate PR and then the output will be generated.

Reply:
so you want all the screenshots removed across the how to guides, correct?

Reply:
Just for this how to guide for now I think since it's very simple for the other one I'm not sure yet.

Reply:
done